sdk/openai/openai/samples-dev/completionsWithContentFilter.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to get completions for the provided prompt and parse output for content filter result.
+ *
+ * @summary get completions.
+ * @azsdk-weight 100
+ */
+
+import { OpenAIClient, AzureKeyCredential, ContentFilterResult } from "@azure/openai";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// You will need to set these environment variables or edit the following values
+const endpoint = process.env["ENDPOINT"] || "<endpoint>";
+const azureApiKey = process.env["AZURE_API_KEY"] || "<api key>";
+
+const prompt = ["What is Azure OpenAI?"];
+
+export async function main() {
+  console.log("== Get completions Sample ==");
+
+  const client = new OpenAIClient(endpoint, new AzureKeyCredential(azureApiKey));
+  const deploymentId = "text-davinci-003";
+  const result = await client.getCompletions(deploymentId, prompt, { maxTokens: 128 });
+
+  for (const choice of result.choices) {
+    console.log(choice.text);
+    if (choice.contentFilterResults) {
```

Comment:
Instead of having nested ifs, can we flip the condition and exit the sample with a message that no content filtering results found?